.github/workflows/release.Hutch.Relay.yml

```diff
@@ -10,30 +10,40 @@ on:
 
 env:
   namespace: hutchstack
-  repo-name: hutch-relay
+  image-name: hutch-relay
+  repo-name: ${{ github.repository }}
```

Comment:
Doesn't look like you're using this variable

Reply:
Sorry this doesn't make it clear. I mean the `repo-name` one

Reply:
repo-name and image-name are used on line 46

Reply:
Oh I see now. I got distracted by the diff highlights lol